### ISSUE TITLE
Remove includes pattern from dprint init

### ISSUE THREAD

**Issue by dsherret** [dprint/dprint#728]
As title. It's not necessary anymore.

**Comment by doox911** [comment]
> As title. It's not necessary anymore.

Hello. But why?

**Comment by dsherret** [comment]
@doox911 the CLI now handles automatically creating an efficient glob when one is not specified based on the provided plugins.

**Comment by doox911-opensource** [comment]
> @doox911 the CLI now handles automatically creating an efficient glob when one is not specified based on the provided plugins.

Sorry for my illiteracy, but I don't understand your answer. Could you describe in more detail or provide a source explaining the mechanism?

**Comment by dsherret** [comment]
@doox911-opensource are you encountering an issue using dprint without an "includes" pattern? If so, could you open an issue? You can still provide one if you'd like, but it's not necessary anymore.

**Comment by doox911-opensource** [comment]
> @doox911-opensource are you encountering an issue using dprint without an "includes" pattern? If so, could you open an issue? You can still provide one if you'd like, but it's not necessary anymore.

No. But I don't understand, what is this **glob**?

**Comment by dsherret** [comment]
Glob is another term for a file pattern. It's used for matching files. For example, if you use dprint-plugin-json and no "includes" pattern is specified, then it will create a pattern that looks like `**/*.{json,jsonc}`, which essentially means "match all files in all descendant directories that have a `.json` or `.jsonc` extension.

**Comment by doox911-opensource** [comment]
> Glob is another term for a file pattern. It's used for matching files. For example, if you use dprint-plugin-json and no "includes" pattern is specified, then it will create a pattern that looks like `**/*.{json,jsonc}`, which essentially means "match all files in all descendant directories that have a `.json` or `.jsonc` extension.

And if I want a file type, for example .some_type, containing json to be formatted, then I have to specify it in includes?

**Comment by dsherret** [comment]
> And if I want a file type, for example .some_type, containing json to be formatted, then I have to specify it in includes?

For that you would need to associate the file extension with the plugin. You can do that by specifying an "associations" pattern on the plugin:

```
{
  "json": {
    "associations": "**/*.{json,jsonc,some_type}"
  },
  "plugins": [
    "https://plugins.dprint.dev/json-0.17.4.wasm"
  ]
}
```

https://dprint.dev/config/#associations

**Comment by doox911-opensource** [comment]
> > And if I want a file type, for example .some_type, containing json to be formatted, then I have to specify it in includes?
> 
> For that you would need to associate the file extension with the plugin. You can do that by specifying an "associations" pattern on the plugin:
> 
> ```
> {
>   "json": {
>     "associations": "**/*.{json,jsonc,some_type}"
>   },
>   "plugins": [
>     "https://plugins.dprint.dev/json-0.17.4.wasm"
>   ]
> }
> ```
> 
> https://dprint.dev/config/#associations

Thanks a lot! That is, the `exclude` property is deprecated and was left only for compatibility? Will it be deleted next?
